src/AbstractEnum.php

```diff
@@ -70,6 +72,28 @@
     }
 
     /**
+     * @param callable|null $callback
+     * @param string|null   $separator
+     *
+     * @return array|\string[]
+     */
+    final public static function getClassPrefixedKeys($callback = null, $separator = null)
+    {
+        $separator = $separator ?: '_';
```

Comment:
This was to avoid people having to specify the default if they want to add another parameter.

But now separator is the last argument. I think this can be reverted.